README.md

```diff
@@ -10,9 +10,18 @@ Basic usage:
     import pandas as pd
```

Comment:
Cam we add `import numpy as np` (necessary now because of the `np.random.rand` that was added)?